synapse/third_party_rules/access_rules.py

```diff
@@ -74,6 +74,11 @@ class RoomAccessRules(object):
             # using the /info endpoint. Required.
             id_server: "vector.im"
 
+            # Enable freezing a room when the last room admin leaves.
+            # Note that the departing admin must be a local user in order for this feature
+            # to work.
+            freeze_room_with_no_admin: false
```

Comment:
This will now need to be added to the module config in the room freezing tests: https://github.com/matrix-org/synapse-dinsic/blob/5c51806d1d3ca767e252bcaf5d6edd25976493f8/tests/rest/client/test_room_access_rules.py#L50-L56